mlir/test/Target/Cpp/emitc-constant-propagation.mlir

```diff
@@ -0,0 +1,19 @@
+// RUN: mlir-translate -mlir-to-cpp -propagate-constants %s | FileCheck %s -check-prefix=CPP-DEFAULT
```

Comment:
Please also update the `mlir/test/Target/Cpp/emitc-constant-propagation.mlir` to reflect the new option's name (see other comment).

Reply:
Done